index.bs

```diff
@@ -66,6 +66,7 @@ namespace console { // but see namespace object requirements below
   undefined time(optional DOMString label = "default");
   undefined timeLog(optional DOMString label = "default", any... data);
   undefined timeEnd(optional DOMString label = "default");
+  undefined timeStamp(optional DOMString label = "default");
```

Comment:
im not sure exactly the behavior of Chrome/Firefox/etc., but at least in WebKit there is no `= "default"` value (i.e. calling `console.timeStamp()` will show "Timestamp" in the UI)

Reply:
Yeah, Chromium appears to not use "default" at all. At least when provided with no string, it shows `"TimeStamp: <empty string technically here>"` in the UI which I think matches Safari, but I fear I am too dumb to figure out where in the "Timelines" UI calls to `console.timeStamp()` actually show up.

<img width="612" alt="Screenshot 2024-07-21 at 11 09 01 PM" src="https://github.com/user-attachments/assets/863e94b3-57be-48f2-b3a8-75fbcc767a83">

Reply:
Yes, I just fixed V8 last week to have the `= "default"` value for all `console.time*()` APIs, including `console.timeStamp()`. Maybe we can update JSC as well to have consistency.

Reply:
i kinda prefer it showing "Timestamp" in the UI, but i dont feel too strongly about it

we could always have the API use `= "default"` and then in the UI check if `=== "default"` and if so show "Timestamp"

Reply:
Ok, so we can stick to `= "default"` for consistency here?

Reply:
> we could always have the API use = "default" and then in the UI check if === "default" and if so show "Timestamp"

This seems a little funky, especially since Safari treats `console.timeStamp()` differently from `console.timeStamp("default")`—the former defaults to "TimeStamp" and the latter prints "default". Whereas Chrome shows "TimeStamp: default" for both. I think it makes more sense to stick with `= "default"` personally. Do you think Safari's inspector would be willing to budge on this @dcrousso ?

Reply:
like i mentioned, im not a huge fan of showing "default" in the UI as i think it's not super clear (especially when compared to showing "Timestamp"), but Web Inspector certainly can adopt the behavior of `DOMString label = "default"` and then have the UI adjust based on whether a value was actually provided